tabbycat/users/forms.py

```diff
@@ -34,7 +34,8 @@ def get_users(self, email):
                 'username': email.split("@")[0],
             },
         )
-        user.membership_set.create(group=self.cleaned_data['role'])
+        if created:
```

Comment:
With the `.add()` method call, do we need this check? It could cause problems if re-inviting a user, but with a new group (like CA + Equity or in another tournament)